pos_hide_partner_info/static/src/xml/templates.xml

```diff
@@ -0,0 +1,53 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<templates id="template">
+    <t t-extend="ClientListScreenWidget">
+
+        <t t-jquery="th:contains('Address')" t-operation="attributes">
```

Comment:
I don't think it will work with translations unfortunatly :/

Reply:
It does I think, I tested it on a DB with french translation and it works)

Reply:
I ask for it to be tested in the functional tests